resources/definitions/raise3D_N2_dual.def.json

```diff
@@ -0,0 +1,95 @@
+{
+    "id": "raise3D_N2_dual",
+    "version": 2,
+    "name": "Raise3D N2 Dual",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Raise3D",
+        "manufacturer": "Raise3D",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "raise3D_N2_dual_extruder_0",
+            "1": "raise3D_N2_dual_extruder_1"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Raise3D N2 Dual" },
+        "machine_width": {
+            "default_value": 305
+        },
+        "machine_height": {
+            "default_value": 305
+        },
+        "machine_depth": {
+            "default_value": 305
+        },
+        "machine_center_is_zero": {
+            "default_value": false
+        },
+        "machine_heated_bed": {
+            "default_value": true
+    	},
+        "machine_nozzle_size": {
+            "default_value": 0.4
+        },
+        "machine_nozzle_heat_up_speed": {
+            "default_value": 6
+        },
+        "machine_nozzle_cool_down_speed": {
+            "default_value": 4
+        },
+        "machine_head_with_fans_polygon":
+        {
+            "default_value": [
+                [ -75, 35 ],
+                [ -75, -18 ],
+                [ 18, 35 ],
+                [ 18, -18 ]
+            ]
+        },
+        "machine_min_cool_heat_time_window": {
+            "default_value": 3600
```

Comment:
Huh, so you basically never want it to go to stand-by temperature, right? Why did you do this? Seems like 5 minutes should be plenty for any PID regulator.